vault/token_store_test.go

```diff
@@ -832,6 +832,35 @@ func TestTokenStore_HandleRequest_CreateToken_NonRoot_InvalidSubset(t *testing.T
 	}
 }
 
+func TestTokenStore_HandleRequest_CreateToken_NonRoot_RootChild(t *testing.T) {
+	_, ts, _, root := TestCoreWithTokenStore(t)
+	testMakeToken(t, ts, root, "sudoClient", "", []string{"create", "update", "sudo"})
```

Comment:
Note that this `[]string` as the last argument is a list of policies, not capabilities. This will work just fine for this test, but looks like a point of confusion.